opm/autodiff/StandardWellsDense.hpp

```diff
@@ -69,7 +72,9 @@ enum WellVariablePositions {
         template<typename FluidSystem, typename BlackoilIndices>
         class StandardWellsDense {
         public:
-
+            // --------- Ebos property system stuff -------
+            typedef typename TTAG(EclFlowProblem) TypeTag;
+            typedef typename GET_PROP_TYPE(TypeTag, ElementContext)    ElementContext;
```

Comment:
if you really want, you can. (provided that you add `TTAG()` around `EclFlowProblem`.)

IMO using `GET_PROP_TYPE(TypeTag, ElementContext)` is more readable, though. Also, the `TypeTag` usually not hard-coded but a template argument.